tests/acceptance/features/webUISharingInternalGroupsToRoot/shareWithGroups.feature

```diff
@@ -147,22 +147,6 @@ Feature: Sharing files and folders with internal groups
     Then file "lorem.txt" should be listed on the webUI
     And as "Carol" the content of "new-simple-folder/lorem.txt" should be the same as the local "lorem.txt"
 
-  @skip @yetToImplement
-  Scenario: user tries to share a file in a group which is excluded from receiving share
-    Given user "Alice" has created file "lorem.txt"
-    And group "system-group" has been created
-    And the administrator has browsed to the admin sharing settings page
-    When the administrator excludes group "system-group" from receiving shares using the webUI
-    Then user "Alice" should not be able to share file "lorem.txt" with group "system-group" using the sharing API
-
-  @skip @yetToImplement
-  Scenario: user tries to share a folder in a group which is excluded from receiving share
-    Given user "Alice" has created folder "simple-folder"
-    And group "system-group" has been created
-    And the administrator has browsed to the admin sharing settings page
-    When the administrator excludes group "system-group" from receiving shares using the webUI
-    Then user "Alice" should not be able to share folder "simple-folder" with group "system-group" using the sharing API
-
```

Comment:
Should cover by https://github.com/owncloud/web/blob/86f35b6018803295aa0b00b23dbbd8de1ac49a54/tests/acceptance/features/webUISharingInternalGroupsToRoot/shareWithGroups.feature#L197